ui/app/components/secure-variable-form.js

```diff
@@ -16,6 +21,20 @@ export default class SecureVariableFormComponent extends Component {
     return !this.args.model?.path;
   }
 
+  @action
+  validatePath(e) {
+    const value = trimPath([e.target.value]);
+    let existingVariable = this.store
+      .peekAll('variable')
```

Comment:
`question` `thought` `non-blocking`:  Can you humor me?

What happens if there are 2 co-workers and they're both creating a variable within minutes of one another? There's no fetch request here to pick up on this.

Should we consider this scenario and use `findAll` instead?

Reply:
As it stands, an error will happen from the API, and get passed up as a "The variable you are trying to create already exists" to the client.

I'll need to pass that on to the user as part of form validation. But one's work wouldn't override the other's.